qcodes/instrument_drivers/american_magnetics/AMI430_VISA.py

```diff
@@ -0,0 +1,102 @@
+# if this actually works...
```

Comment:
I think the IPToVisa class should be placed in a testing utilities module. Then for every driver test that is not a VISA instrument, generate a VISA equivalent using this class, write a yaml file to generate a mock instrument and write your tests by instatiating a driver class.

Reply:
You are absolutely right, there is nothing AMI430-specific to that class. I'll move it.